src/main/java/com/redhat/red/build/finder/report/BuildStatisticsReport.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (C) 2017 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.redhat.red.build.finder.report;
+
+import java.util.List;
+
+import com.redhat.red.build.finder.BuildStatistics;
+import com.redhat.red.build.finder.KojiBuild;
+
+public class BuildStatisticsReport extends Report {
+    private BuildStatistics buildStatistics;
+
+    public BuildStatisticsReport(List<KojiBuild> builds) {
+         buildStatistics = new BuildStatistics(builds);
+    }
+
+    public BuildStatistics getBuildStatistics() {
+        return buildStatistics;
+    }
+
+    @Override
+    public String render() {
+        return String.format("%d %d %f %d %d %f\n", buildStatistics.getNumberOfBuilds(), buildStatistics.getNumberOfImportedBuilds(), buildStatistics.getPercentOfBuildsImported(), buildStatistics.getNumberOfArchives(), buildStatistics.getNumberOfImportedArchives(), buildStatistics.getPercentOfArchivesImported());
```

Comment:
I think at the moment this statistics isn't helpful unless we can categorise the columns. If I have e.g. `1 0 0.000000 2 0 0.000000` I don't know what that means.

Reply:
I thought that the general idea behind the text output was that it is supposed to be machine readable, suitable for (eventual) passing to other tools. It may be possible to put a comment header or something like that.

However, I really intended every report to also output HTML, and then the HTML report would contain the human-readable/pretty format such as tables, etc.

For example, we could have a `renderHTML()` method that outputs a proper HTML table with headers, and then this is what would get included in the HTML report (or a separate HTML file).

Reply:
Maybe we can add an initial line to each .txt file with a list of columns?

gav.text
# Maven Group-Artifact-Version

nvr.text
# Koji Name-Version-Release

statistics.text
# Total builds    Builds imported    Imports %    Total artifacts    Artifacts imported  Imports %

Reply:
Whatever we decide for the text, I'd like to submit the HTML reports as a separate pull request.

The HTML reports will be a table-formatted version of what's in the text files right now. This data can be included in a single page along with the current builds table HTMLReport, or as separate HTML files, or both.

Reply:
I like @jcoleman0redhat0com  suggestion with a header line (can we put separators (e.g. ';' ) in the header line through as currently its hard to read )

+1 to separate pr for for reports.

Reply:
I think it only makes sense to use the same separator as in the data itself, though.

Reply:
My point is I can't tell in @jcoleman0redhat0com which is part of which column as the words are space separated. I don't really care how its clarified .... even putting the columns together with camel case but it should be understandable.